### DISCUSSION TITLE
Should we release 0.72.10?

### DISCUSSION THREAD

**Discussion by huntie** [reactwg/react-native-releases#97] [Patches]
# Should we release 0.72.10?

Current Release: [0.72.9](https://github.com/facebook/react-native/releases/tag/v0.72.9)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Local commits to backport to main

1.

**Comment by joe-sam** [comment]
Corrects LogBox to display the actual stack trace libraries instead of internal code on the debug thread for all custom _non-standard_ errors during unhandled promises. JavaScript allows to use throw with any argument, so technically our custom error classes don’t need to inherit from Error (e.g. third-party libraries).

https://github.com/facebook/react-native/pull/42079

**Reply by huntie** [reply]
Picked ✅

**Comment by cipolleschi** [comment]
Fix for `symbol not found _jump_fcontext with use_frameworks!` (#39945):
* https://github.com/facebook/react-native/pull/42228

**Reply by huntie** [reply]
Merged ✅

**Comment by cipolleschi** [comment]
Fix min ios version regression and updates podfile.lock: 
* https://github.com/facebook/react-native/pull/42232

**Reply by huntie** [reply]
Applied locally (`Podfile.lock` conflicts) ✅

**Comment by cipolleschi** [comment]
Disable bitcode for hermes: https://github.com/facebook/react-native/pull/42142

**Reply by huntie** [reply]
Merged ✅

**Comment by kirillzyusko** [comment]
Would be good to include https://github.com/facebook/react-native/pull/42075 👀 

It wasn't a problem to use `install_modules_dependencies` in RN 0.71 and in RN 0.73, but 0.72 can not use `install_modules_dependencies` (if library is using Swift code) and PR from above solves the problem (I hope it can be treated as qualified pick).
